src/template-functions.php

```
@@ -567,3 +567,141 @@ function ( $date ) use ( $post ) {
+if ( ! function_exists( 'se_modify_event_date_archive_body_class' ) ) {
+	/**
+	 * Modify the body class for the event date archive.
+	 *
+	 * @param array $classes The existing body classes.
+	 *
+	 * @return array The modified body classes.
+	 */
+	function se_modify_event_date_archive_body_class( $classes ) {
+		$classes = array_map(
+			function ( $body_class ) {
+				return 'post-type-archive-se-event-date' === $body_class ? 'post-type-archive-se-event' : $class;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$class'. (Unused Code Rules)

(UnusedLocalVariable)